config/toml.go

```diff
@@ -330,6 +331,7 @@ max-connections = {{ .P2P.MaxConnections }}
 max-incoming-connection-attempts = {{ .P2P.MaxIncomingConnectionAttempts }}
 
 # List of node IDs, to which a connection will be (re)established ignoring any existing limits
+# TODO: Remove once p2p refactor is complete.
```

Comment:
Is there a tracking issue # we can add here for that work? I'm not sure if we use this convention, but I find it helpful to use a format like `TODO(#xxxx): Blah blah do the thing` so the reader can figure out the current status easily.

Reply:
Yeah we usually do. It's the same reference issue that's used above. I've added it here for completeness